src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Todo.css';
+
+ const Todo = ({item}) =>{
```

Comment:
this is a clever use of destructuring, but I would still recommend taking in props as normal just for the sake of clarity in your code.